back/src/Guide/Api/Types.hs

```diff
@@ -196,6 +197,15 @@ data ItemSite route = ItemSite
       :> "item"
       :> Capture "id" (Uid Item)
       :> Delete '[JSON] NoContent
+
+  , _moveItem :: route :-
+      Summary "Move item"
+      :> ErrorResponse 404 "Item not found"
+      :> "item"
+      :> Capture "item" (Uid Item)
+      :> "move"
+      :> Capture "type" Move
+      :> Put '[JSON] NoContent
```

Comment:
This should be a `Post`, not `Put`. The rule of thumb is that a `Put` request should always be safe to repeat, i.e. doing a `Put` request twice should be the same as doing it once.